WORKSPACE

```diff
@@ -475,15 +479,14 @@ container_pull(
 container_pull(
     name = "fedora_with_test_tooling_ppc64le",
     digest = "sha256:9ec3e137bff093597d192f5a4e346f25b614c3a94216b857de0e3d75b68bfb17",
-    puller_linux = "@go_puller_linux_ppc64le//file:downloaded",
+    #    puller_linux = "@go_puller_linux_ppc64le//file:downloaded",
```

Comment:
I think you can simply remove these lines.

Reply:
Updated